modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/paths/TestrayManage.path

```diff
@@ -25,6 +25,11 @@
 	<td>//input[contains(@id, 'name') and contains(@type, 'text')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>NEW_OPTION_IN_MODAL</td>
+	<td>//div[@class='modal-body']/..//input[@id='name']</td>
```

Comment:
Would you please check if //div[@class='modal-body']//input[@id='name'] works with it? It looks like the "/.." is redundant here

Reply:
Yes, it works